aiosmtpd/context.py

```diff
@@ -0,0 +1,15 @@
+class SessionContext:
+    peer = None
+    ssl = None
+    host_name = None
+    extended_smtp = False
+
+
+class MessageContext:
```

Comment:
While get its name I was thinking about my project where this entity is moving to be connected with email.Message, but was aware of using same name. Just found that email protocol have special term "envelope" for that.